pkg/util/util.go

```diff
@@ -255,9 +258,9 @@ func ParseMetric(rawName string) (string, string, string, string) {
 		name, metricType = ParseMetricType(name)
 	} else {
 		name = rawName
-		display = strings.ReplaceAll(rawName, ".", "_")
-		display = strings.ReplaceAll(display, "-", "_")
+		display = rawName
 	}
+	display = displayReplacer.Replace(display)
```

Comment:
This also need to be changed: https://github.com/NetApp/harvest/blob/047dd568d14987fbbe75daa6fcdb8d419684e465/cmd/collectors/zapiperf/zapiperf.go#L857 and line 1058

RestPerf as well.